src/github.com/getlantern/flashlight/main/main.go

```diff
@@ -266,7 +265,6 @@ func afterStart(cfg *config.Config) {
 
 func onConfigUpdate(cfg *config.Config) {
 	autoupdate.Configure(cfg)
-	proxiedsites.Configure(cfg.ProxiedSites)
```

Comment:
Right, but we still use proxied sites as a general thing in Lantern @xiam! This change makes it so those aren't configured from the cloud config.